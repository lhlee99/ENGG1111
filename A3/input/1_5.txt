110001110001001010100111011101100101001011010111
10111000100110001010001000110110010111000001111
