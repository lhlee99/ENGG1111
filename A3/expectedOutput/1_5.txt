1001000110101111011111000100100011000000011100110
